packages/taco/src/conditions/base/context-variable.ts

```diff
@@ -0,0 +1,22 @@
+import { Condition } from '../condition';
+import {
+  ContextVariableConditionProps,
+  contextVariableConditionSchema,
+  ContextVariableConditionType,
+} from '../schemas/context-variable';
+import { OmitConditionType } from '../shared';
+
+export { ContextVariableConditionProps, ContextVariableConditionType };
+
+/**
+ * A condition that performs comparison operations on context variable values.
+ * Supports various comparison operators like ==, >, <, >=, <=, !=, in, !in.
+ */
```

Comment:
Minor nitpick - I wouldn't list the operators. You can accept or reject.

```suggestion
/**
 * A condition that performs comparison operations on context variable values.
 */
```